src/sentry/static/sentry/app/views/eventsV2/results.tsx

```diff
@@ -347,9 +365,12 @@ class Results extends React.Component<Props, State> {
               organization={organization}
               location={location}
               eventView={eventView}
+              handleIncompatibleQuery={this.handleIncompatibleQuery}
+              handleIncompatibleAlertClose={this.handleIncompatibleAlertClose}
```

Comment:
The pattern we use is `on*` prefix for prop names and `handle*` for the callback handler.

Reply:
ah got confused since the prop is passed down a level